src/components/Hero/Hero.tsx

```diff
@@ -48,84 +40,137 @@ export default function Hero(props: React.PropsWithChildren<HeroProps>) {
+    console.warn(
+      `Warning: it is recommended to use both "backgroundImageSrc" and "image" props for CAMPAIGN hero`
+    );
+    modifiers.push("warning");
   }
 
   if (heroType !== HeroTypes.Primary && locationDetails) {
-    throw new Error(`Please provide locationDetails only to PRIMARY heroTypes`);
+    console.warn(
+      `Warning: Please provide "locationDetails" only to PRIMARY hero`
+    );
+    modifiers.push("warning");
   }
 
-  const backgroundImageStyle = backgroundImageSrc
-    ? { backgroundImage: "url(" + backgroundImageSrc + ")" }
-    : { backgroundColor: backgroundColor };
+  let backgroundImageStyle = {};
+  if (heroType === HeroTypes.Primary) {
+    backgroundImageStyle = backgroundImageSrc
+      ? { backgroundImage: "url(" + backgroundImageSrc + ")" }
+      : null;
+  } else if (
+    heroType !== HeroTypes.Secondary &&
+    heroType !== HeroTypes.FiftyFifty
+  ) {
+    backgroundImageStyle = backgroundImageSrc
+      ? { backgroundImage: "url(" + backgroundImageSrc + ")" }
+      : { backgroundColor: backgroundColor };
+  }
 
   let contentBoxStyling = {};
   if (
     heroType === HeroTypes.Primary ||
     heroType === HeroTypes.Tertiary ||
-    heroType === HeroTypes.FiftyFifty
+    heroType === HeroTypes.FiftyFifty ||
+    heroType === HeroTypes.Campaign
   ) {
-    if (foregroundColor && backgroundColor) {
-      contentBoxStyling = {
-        color: foregroundColor,
-        backgroundColor: backgroundColor,
-      };
-    } else if (foregroundColor || backgroundColor) {
-      const receivedColor = foregroundColor
-        ? "foregroundColor"
-        : "backgroundColor";
-      throw new Error(
-        `Please provide both foregroundColor and backgroundColor to Hero, only got ` +
-          receivedColor
-      );
-    }
+    contentBoxStyling = {
+      color: foregroundColor,
+      backgroundColor: backgroundColor,
+    };
   } else {
     if (foregroundColor || backgroundColor) {
-      throw new Error(
-        `Received foregroundColor and/or backgroundColor, but these are only accepted on HeroTypes.Primary and HeroTypes.FiftyFifty`
+      console.warn(
+        `Warning: the "foregroundColor" and/or "backgroundColor" props have been passed, but SECONDARY Hero will not use them`
       );
+      modifiers.push("warning");
     }
   }
 
   return (
     <div
-      className={bem(heroBaseClass, heroModifiers, blockName, [className])}
+      className={bem(heroBaseClass, modifiers, blockName, [className])}
       data-responsive-background-image
       style={backgroundImageStyle}
     >
-      <div
-        className={bem("content", [], heroBaseClass)}
-        style={contentBoxStyling}
-      >
-        {heading}
-
-        {image}
-
-        {subHeaderText}
-      </div>
+      {heroType === HeroTypes.Campaign ? (
+        <div
+          className={bem("content", [], heroBaseClass)}
+          style={contentBoxStyling}
+        >
```

Comment:
This whole block is good but you're repeating the `<div clasName={bem.....` section twice. What you can do is keep that as is and only update the internal DOM by either 1) moving the logic check on line 147 down or 2) declare that logic check above and assign the child blocks to a variable. So something like
```
const childrenToRender = heroType === HeroTypes.Campaign ? <>{image}.....</> : <>{heroType.....}</>;
```

Reply:
That's actually how I had it initially.  That formatting need some type of wrapper around the children (`<div>` or `<>`), so I wrote it that way that I did to remove unneeded tags.  However, now I see that using `<>` does not insert extra tags.  That being the case, I went that route and dropped all the logic into a variable assignment.